templates/_header.html

```diff
@@ -21,10 +21,29 @@
           </a>
         </li>
         <li class="p-navigation__link" role="menuitem"><a href="https://build.snapcraft.io">Build</a></li>
-        <li class="p-navigation__link" role="menuitem"><a class="p-link--external" href="https://dashboard.snapcraft.io/snaps">My snaps</a></li>
+        <li class="p-navigation__link" role="menuitem"><a href="/account">My snaps</a></li>
```

Comment:
`/account` redirects to `/account/snaps` but can you use `/account/snaps` to avoid problem in the future

Reply:
Forget it, I mixed this PR with the split of the pages :)